frontend/src/components/sectionButton.tsx

```diff
@@ -23,7 +23,7 @@ const SectionButton: React.FC<SectionButtonProps> = ({ color = "red", title, sub
 
   // Function to toggle dropdown visibility
   const toggleDropdown = () => {
-    setIsDropdownVisible(!isDropdownVisible);
+    setIsDropdownVisible(false);
```

Comment:
Since we're not implementing the dropdowns anymore, can you remove the `isDropdownVisible` completely? And just not render the hamburger button at all since it doesn't do anything? And vertically center the text in the section?
<img width="1080" height="2424" alt="Screenshot_20251116-145324" src="https://github.com/user-attachments/assets/34b122cd-4993-4033-9280-551c3af102f9" />